base_view_inheritance_extension/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2016 Therp BV <https://therp.nl>
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Extended view inheritance",
+    "version": "14.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "category": "Hidden/Dependency",
+    "summary": "Adds more operators for view inheritance",
+    "website": "https://github.com/OCA/server-tools",
+    "depends": ["base"],
```

Comment:
from #1946 
```suggestion
    "depends": ["base"],
    "external_dependencies": {"python": ["pyyaml"]},
```

Reply:
@alexis-via this is marked as resolved but the travis error is still there. Was it lost during a rebase?

(marking it as unsolved)